### ISSUE TITLE
tracking.es6:18: Move LoadedGATracker to the refarm.

### ISSUE THREAD

**Issue by 0pdd** [fidals/shopelectro#761]
The puzzle `759-bc6cfc4f` from #759 has to be resolved: 

https://github.com/fidals/shopelectro/blob/987327e514575142670d965adc25bc3203bf9f8d/front/js/shared/tracking.es6#L18-L18

The puzzle was created by Artemiy on 07-Mar-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0pdd** [comment]
@artemiyds2 @duker33 please pay attention to this new issue.

**Comment by artemiy312** [comment]
It is blocked by https://github.com/fidals/shopelectro/issues/762

**Comment by 0pdd** [comment]
@0pdd the puzzle [#795](https://github.com/fidals/shopelectro/issues/795) is still not solved.
